components/DataSectionSummary.tsx

```diff
@@ -18,24 +18,27 @@ export const DataSectionSummary = ({ rows }: DataSectionSummaryProps) => {
         height: '100%',
       }}
     >
-      {PRIORITY_TYPES.map((priority) => (
-        <PriorityBadge
-          key={priority}
-          variant="count"
-          priority={priority}
-          totalCount={
-            rows.filter((row: GridValidRowModel) => row.priority === priority)
-              .length
-          }
-          completedCount={
-            rows.filter(
-              (row: MeasureRow) =>
-                row.priority === priority &&
-                measureTemplateHasValue(row.originalMeasureTemplate)
-            ).length
-          }
-        />
-      ))}
+      {PRIORITY_TYPES.map((priority) => {
+        const totalCount = rows.filter(
+          (row: GridValidRowModel) => row.priority === priority
+        ).length;
+
+        const completedCount = rows.filter(
+          (row: MeasureRow) =>
+            row.priority === priority &&
+            measureTemplateHasValue(row.originalMeasureTemplate)
+        ).length;
+
+        return totalCount > 0 || completedCount > 0 ? (
+          <PriorityBadge
+            key={priority}
+            variant="count"
+            priority={priority}
+            totalCount={totalCount}
+            completedCount={completedCount}
+          />
+        ) : null;
+      })}
```

Comment:
Does this affect the priorities being aligned? For example in the screenshot below, high priority counts are still aligned vertically even if there aren't any medium or low ones

![Screenshot 2024-08-19 at 10 32 25](https://github.com/user-attachments/assets/a12d2a31-e405-4e7a-ade9-cdf08fd4f2d9)